test/harness/index.js

```diff
@@ -134,7 +141,7 @@ function assert_invalid(bytes) {
         } catch(e) {
             assert_true(e instanceof WebAssembly.CompileError, "expected invalid failure:");
         }
-    }, "A wast module that should be invalid or malformed.");
+    }, testNum() + "A wast module that should be invalid or malformed.");
```

Comment:
Instead of doing this at every call site to `test`, why not call a wrapper that does it?